client/src/components/Subscription/index.tsx

```diff
@@ -0,0 +1,365 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { TUser, useGetUserByIdQuery } from '@librechat/data-provider';
+import { useRecoilValue } from 'recoil';
+import store from '~/store';
+import { localize } from '~/localization/Translation';
+import Cookies from 'js-cookie';
+import { v4 as uuidv4 } from 'uuid';
+import { loadStripe } from '@stripe/stripe-js';
+
+const STRIPE_PUBLIC_KEY = import.meta.env.VITE_STRIPE_PUBLIC_KEY
+
+function SubscriptionContent() {
+  const [darkMode, setDarkMode] = useState(window.matchMedia('(prefers-color-scheme: dark)').matches);
+  const [subscriptionUser, setSubscriptionUser] = useState<TUser | null>(null);
+  const [error, setError] = useState('');
+  const { userId = '' } = useParams();
+  const lang = useRecoilValue(store.lang);
+  const navigate = useNavigate();
+  const [subscriptionDueDate, setSubscriptionDueDate] = useState('');
+  const [subscribedPlanId, setSubscribedPlanId] = useState('');
+
+  const getUserByIdQuery = useGetUserByIdQuery(userId);
+
+  const subscriptionOptions = [
+    {
+      planId: 'month',
+      price: '120.00',
```

Comment:
Is that possible to change the price from a string to a number here too?

Reply:
> Is that possible to change the price from a string to a number here too?

updated